frontend/src/App.js

```diff
@@ -4,7 +4,9 @@ import Calendar from './Calendar.js';
 
 function App() {
   return (
-    <Calendar />
+    <div>
+      <Calendar />
+    </div>
```

Comment:
Is there a reason for this div? It doesn't seem to do anything here.

Reply:
True, I used to have buttons in there, but forgot to remove the <div> when I got rid of them.